app/forms/questionnaire_form.py

```diff
@@ -122,13 +122,17 @@ def validate_mutually_exclusive_question(self, question):
             question["validation"].get("messages") if "validation" in question else None
         )
         answers = (getattr(self, answer["id"]).data for answer in question["answers"])
+        only_checkboxes = all(
+            answer["type"] == "Checkbox" for answer in question["answers"]
+        )
```

Comment:
Can we make this consistent with `is_mandatory` and call this `is_only_checkboxes` or `are_all_checkboxes`?